web/cobrands/centralbedfordshire/flytipping.js

```diff
@@ -0,0 +1,55 @@
+document.addEventListener('category_extra_loaded', function() {
```

Comment:
Or, a second alternative would be to have permanent listeners set up here on a parent that's always present (document, or the form, etc) e.g. `$(document).on('click', '#form_fly_tip_witnessed')` which wouldn't then need to be run on every category change.